### PR TITLE
vNext documentation updates

### PR DESCRIPTION
**Pull request by davidkline-ms**
Add roadmap for vNext releases through the end of 2018.

### CODE REVIEW

File: Roadmap.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+Release details, including backlog items, can be found on the [GitHub project pages](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects).
+
+# Mixed Reality Toolkit (MRTK) Roadmap
+
+The Mixed Reality Toolkit (also known as "vNext") is an all-new product, built to be cross MR/AR/VR/XR platform by design. There are two planned pre-releases after which the Mixed Reality Toolkit will become the primary product.
+
+The Mixed Reality Toolkit will require Unity 2018 or newer.
+
+| When Unity releases an LTS (Long Term Support) product, the Mixed Reality Toolkit will branch (ex: mrtk_2018_LTS) and require an LTS release. This branch will be targeted at developers who wish to have the most stable MRTK on which to build. It is expected that the branched Mixed Reality Toolkit will receive bug fixes and no new features. The mrtk_development branch will update to the next Unity Tech release (ex: 2019.1) and **may** drop backward compatibility.
```
Comment (SimonDarksideJ) [inline Roadmap.md]: Is the "|" meant to be a ">" character to denote a blockquote?
Change or remove.

File: Roadmap.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- Windows Mixed Reality
+    - Microsoft HoloLens
+    - Immersive Headsets
+- OpenVR
+
+## August 2018 - Beta
+
+The MRTK Beta release will introduce initial support for modularization. It is also when the vast majority of HoloToolkit features will be added. Supported platforms will be unchanged from the Alpha release.
+
+| It is important to note that the goal is feature parity and **not** zero app code change. There will be consolidation of duplicated functionality as well as namespace, component and API changes.
```
Comment (SimonDarksideJ) [inline Roadmap.md]: Is the "|" meant to be a ">" character to denote a blockquote?
Change or remove.

File: Roadmap.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+## July 2018 - Alpha
+
+The Mixed Reality Toolkit (MRTK) Alpha, released in late July 2018 is being released as a single package (core, sdk and examples) as the team introduces the new architecture. This release does **not** contain all of the features and functionality of the HoloToolkit releases.
+
+The theme(s) for the Alpha release are
+
+- Minimum Viable Product
+    - New architecture
+    - Render scene
```
Comment (StephenHodgson) [inline Roadmap.md]: What's a render scene?

File: Roadmap.md (hunk 1 above)
Comment (davidkline-ms) [reply Roadmap.md]: It wasn't intended to be a blockquote, but I agree that is a better format for this.

File: Roadmap.md (hunk 2 above)
Comment (davidkline-ms) [reply Roadmap.md]: It wasn't intended to be a blockquote, but I agree that is a better format for this.

File: Roadmap.md (hunk 3 above)
Comment (davidkline-ms) [reply Roadmap.md]: It was supposed to mean that we render scenes... i'll remove it in my next commit.

**Pull request merged**